markdown/articles/2020/12/20/historyofdoomsday.md

```diff
@@ -0,0 +1,1030 @@
+clean pile being available making use of the, as then, new cards from Mirrodin block.
+
+<row variant="pile">{{!Predict}} {{!Brain Freeze}} {{!LED}} {{!CB}} {{!Second Sunrise}}</row>
+
+This cost a cantrip plus {2}{U} to enact on the same turn or you could also pass the turn with it. You cast
+Predict, naming Brain Freeze and drawing LED and Bauble. You cast both, crack LED for {W} and activate 
+Bauble, targeting nothing. Bauble will draw {{Second Sunrise}} which you cast, returning LED and Bauble to play.
+You then crack LED for {W} again and activate Bauble targeting Second Sunrise which you will then draw.
+You repeat this loop until your storm count is high enough and finish by cracking LED for {U} and targeting
+{{Brain Freeze}}. The pile is mana efficient however relied on the graveyard to be used which was dangerous as
+{{Tormod's Crypt}} was one of the most commonly played sideboard cards at the time. It also had some trouble
+finding the pieces as the only non-{{Brainstorm}} cantrips available were {{Serum Visions}} and thing like {{Portent}} 
+however these weren't even really used. The main lists focussed on using Mystical Tutor and {{Lim-dul's Vault}} 
+to assemble the pieces they needed to go off.
+
+When Champion's of Kamigawa came out in late 2004 it brought with it a new tool to the deck in the form of 
+{{Sensei's Divining Top}}. With this being printed a new combo was identified that allowed pilots to avoid
+the use of the graveyard entirely. If you combine a pair of Divining Tops with {{Helm of Awakening}}, you can
+loop them to generate infinite storm as each Top now casts for {0}. You can use these to generate storm and
+then "spin" the top at the end to draw into the lethal Brain Freeze. The pile in isolation can seem clunky
+however as multiple Tops and Helms were run, it is likely you would aready have cost reduction or digging
+already available.
+
+2UUU
+<row variant="pile">{{!IU}} {{!Helm of Awakening}} {{!SDT}} {{!SDT}} {{!Brain Freeze}}</row>
+
+1 Top in play + 2UU
+<row variant="pile">{{!IU}} {{!Helm of Awakening}} {{!LED}} {{!SDT}} {{!Brain Freeze}}</row>
+
+1 Helm in play + UU
+<row variant="pile">{{!IU}} {{!SDT}} {{!LED}} {{!SDT}} {{!Brain Freeze}}</row>
+
+This pile was resilient to graveyard interation however was still reliant on both activated abilities and
+artifact synergies which were hurt by {{Pithing Needle}} and {{Null Rod}} respectively which were relatively
+common from sideboards. The decks were certainly playable, but far from competetive at the time.
+It wasn't truly until a group of people working on completely different decks found the answer did the
+Legacy deck truly form.
+
+
+## The Start of Legacy - Early Storm Lists
+
+Moving sideways for the time being, storm decks were viable from the onset of Legacy with the most well known
+list at the time being known as 'Iggy Pop'. Iggy was a {U}{B} storm list based around the engine of {{Ill-Gotten Gains}}
```

Comment:
I think calling out the creator of Iggy Pop, Michael Bomholt, is appropriate here. His original primer is here (via wayback machine because SCG seems to have lost it and/or I can't find it): https://web.archive.org/web/20090104201512/http://www.starcitygames.com/php/news/article/10989.html